homework.md

```diff
@@ -86,25 +88,25 @@ You send the changes of your local repository to Github
 
 What is Github?
 
-A place for us to store and share our code.
+GitHub is a web-based interface that uses Git, the open source version control software that lets multiple people make separate changes to web pages at the same time. GitHub encourages teams to work together to build and edit their site content.
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+It means a change has been make to the orginal project 
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+Java Script
```

Comment:
Hey, this is the programming language, they are talking about the editor we use to write the code